tests/client/core/test_middleware.js

```diff
@@ -68,17 +70,51 @@ describe('Prefix Middleware', () => {
     assert.deepEqual(fakeRes.set.firstCall.args, ['vary', []]);
   });
 
-  it('should render a 404 when a URL exception is found', () => {
+  it('should render 404 when a localeURL exception exists', () => {
```

Comment:
My point was the test comment makes it sound like a 404 is tied to the localeURL exception - which might be confusing in the future if we ever needed a route without a locale.